### PR TITLE
Update polls.md

### LINKED ISSUE #42: Add documentation for Polls Cog
https://docs.pydrocsid.ml/cogs/general/polls/

### PR DESCRIPTION
**Pull request by ghost**
closes #42

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

File: docs/cogs/general/polls.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+This cog contains the poll commands.
+
+```
+<> - required
+[] - optional
+```
+
+## `yes/no`
+The `.yesno` command puts :thumbsup: :thumbsdown: as reactions under the message (pictures works too).
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
The `.yesno` command puts :thumbsup: :thumbsdown: as reactions under the message (pictures work, too).
```

File: docs/cogs/general/polls.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+```css
+.[yesno|yn] [content]
+```
+If u put a message link behind the command (on the position of the content) the bot puts the reactions on the message from which the link originates
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
If `content` is a message link the bot puts the reactions on the message the link refers to.
```

File: docs/cogs/general/polls.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`content`|       |The message/content|
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
|`content`|       |The message content/A message link|
```

File: docs/cogs/general/polls.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+.[poll|vote] <question>
+[emoji1] <option1>
+[emojiX] [optionX]
+```
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`question`|:heavy_check_mark:|The poll topic/question|
+|`emojiX`|       |The reaction emote under this message|
+|`option1`|:heavy_check_mark:|The poll content for this reaction|
+|`optionX`|       |The poll content for this reaction|
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
|`optionX`|       |Poll option X|
```

File: docs/cogs/general/polls.md [hunk 5]
```diff
@@ -1 +1,54 @@
...
+If u put a message link behind the command (on the position of the content) the bot puts the reactions on the message from which the link originates
+
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`content`|       |The message/content|
+
+
+
+## `poll`
+The `.poll` command makes a poll with 1-19 options.
```
Comment (4norak) [inline docs/cogs/general/polls.md]: Please add documentation for multiline title and options

File: docs/cogs/general/polls.md [hunk 6]
```diff
@@ -1 +1,54 @@
...
+The `.teampoll` command starts a poll and shows, which teamler has not voted yet.
+
+```css
+.[teampoll|teamvote|tp] <question>
+[emoji1] <option1>
+[emojiX] [optionX]
+```
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`team-role`|:heavy_check_mark:|Team-role from the staff|
```
Comment (4norak) [inline docs/cogs/general/polls.md]: This is not an argument for this function or command

File: docs/cogs/general/polls.md [hunk 7]
```diff
@@ -1 +1,54 @@
...
+|`question`|:heavy_check_mark:|The poll topic/question|
+|`emojiX`|       |The reaction emote under this message|
+|`option1`|:heavy_check_mark:|The poll content for this reaction|
+|`optionX`|       |The poll content for this reaction|
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
|`optionX`|       |The poll option for this reaction|
```

File: docs/cogs/general/polls.md [hunk 8]
```diff
@@ -1 +1,54 @@
...
+
+```css
+.[poll|vote] <question>
+[emoji1] <option1>
+[emojiX] [optionX]
+```
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`question`|:heavy_check_mark:|The poll topic/question|
+|`emojiX`|       |The reaction emote under this message|
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
|`emojiX`|       |The reaction emote for option X|
```

File: docs/cogs/general/polls.md [hunk 9]
```diff
@@ -1 +1,54 @@
...
+|`option1`|:heavy_check_mark:|The poll content for this reaction|
+|`optionX`|       |The poll content for this reaction|
+
+
+## `teampoll`
+The `.teampoll` command starts a poll and shows, which teamler has not voted yet.
```
Comment (4norak) [inline docs/cogs/general/polls.md]: Please add documentation for multiline title and options

File: docs/cogs/general/polls.md [hunk 10]
```diff
@@ -1 +1,54 @@
...
+```css
+.[teampoll|teamvote|tp] <question>
+[emoji1] <option1>
+[emojiX] [optionX]
+```
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`team-role`|:heavy_check_mark:|Team-role from the staff|
+|`question`|:heavy_check_mark:|The poll topic/question|
+|`emojiX`|       |The reaction emote under this message|
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
|`emojiX`|       |The reaction emote for option X|
```

**Review by 4norak — CHANGES_REQUESTED** [review]
Inconsistent number of newlines (in this file alone as well as compared to the other documentation files)

**Review by Defelo — DISMISSED** [review]

File: docs/cogs/general/polls.md [hunk 11]
```diff
@@ -1,54 +1,54 @@
...
 ```
-If u put a message link behind the command (on the position of the content) the bot puts the reactions on the message from which the link originates
 
 |Argument|Required|Description|
 |:------:|:------:|:----------|
-|`content`|       |The message/content|
-
+|`content`|       |The message content / A message link|
 
+If `content` is a message link the bot puts the reactions on the message this link refers to.
```
Comment (4norak) [inline docs/cogs/general/polls.md]:
```suggestion
If `content` is a message link, the bot puts the reactions on the message this link refers to.
```

**Review by 4norak — CHANGES_REQUESTED** [review]

**Review by 4norak — APPROVED** [review]

**Pull request merged**